dev-resources/logback-test.xml

```diff
@@ -5,7 +5,8 @@
         </encoder>
     </appender>
 
-    <!--<logger name="log4j.logger.org.eclipse.jetty.server" level="warn"/>-->
+    <logger name="org.eclipse.jetty" level="warn"/>
+    <logger name="org.apache.http" level="warn"/>
```

Comment:
Agree, unrelated, but it's been generally useful for me when debugging tests in many situations.  I made it when debugging these tests and forgot to remove it, but to me it seems worth leaving in.